### ISSUE TITLE
SA1101 false positive on value tuple with named members

### ISSUE THREAD

**Issue by antoinebj** [DotNetAnalyzers/StyleCopAnalyzers#2602]
```csharp
var tuple = (MemberA: "A", MemberB: "B");
```

That code raises two SA1101* warnings, one on MemberA and one on MemberB.

*Prefix local calls with this

**Comment by sharwell** [comment]
Duplicate of #2534 👍
